src/containers/notification/store/actions.ts

```diff
@@ -156,6 +169,8 @@ export type SetNotificationModal = ReturnType<typeof setNotificationModal>
 export type ToggleNotificationPanel = ReturnType<typeof toggleNotificationPanel>
 export type SubscribeUser = ReturnType<typeof subscribeUser>
 export type UnsubscribeUser = ReturnType<typeof unsubscribeUser>
+export type SetPlaylistUpdates = ReturnType<typeof setPlaylistUpdates>
+export type UpdatePlaylistView = ReturnType<typeof updatePlaylistView>
```

Comment:
I would try and rename this `UpdatePlaylistLastViewedAt` and similar throughout. I know it's a pain, but this naming here suggests to me that the "view" of the playlist is changing